src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -155,22 +155,100 @@ public User getUser(String email) {
     return user;
   }
 
-    /**
-     * Gather a list of all users that have entered something into the community chat.
-     *
-     * @return a Set of users, no order to this list, empty list if there are no users that have posted
-     */
-    public Set<String> getUsers () {
-      Set<String> users = new HashSet<>();
-      Query query = new Query("Message");
-      PreparedQuery results = datastore.prepare(query);
-      for (Entity entity : results.asIterable()) {
-        users.add((String) entity.getProperty("user"));
+  /**
+   * Gather a list of all users that have entered something into the community chat.
+   *
+   * @return a Set of users, no order to this list, empty list if there are no users that have posted
+   */
+  public Set<String> getUsers () {
+    Set<String> users = new HashSet<>();
+    Query query = new Query("Message");
+    PreparedQuery results = datastore.prepare(query);
+    for (Entity entity : results.asIterable()) {
+      users.add((String) entity.getProperty("user"));
+    }
+    return users;
+
+  }
+
+  /** Stores a new  Place in Datastore. */
+  public void storePlace(Place place) {
```

Comment:
Can we move these into static methods on place? So `Place.store(Place place)` and `Place.getAll()`.

Reply:
I think a better course of action for now is to see if place works at all with frontend first. Once that's confirmed, we can change all of the different backend data class get/post methods (User, Message, Place, ...) to static so it's consistent across the board. Lmk what you think

Reply:
There is very little risk that moving this into a static method will cause place not to work with frontend. I appreciate how you value consistency, but new code should follow a new standard -- it's easy to forget to clean up code later.